### PR TITLE
Typos doc

### PR DESCRIPTION
**Pull request by lionel-**
I did a bit of doc proofreading.

Also `zip_n()` now takes a `.l` argument instead of `.x`, as with `map_n()` and `cross_n()`.

### CODE REVIEW

File: R/composition.R [hunk 1]
```diff
@@ -170,10 +168,9 @@ lift_vd <- function(..f, ..., .type) {
 #'
 #' @export
 #' @examples
-#' \dontrun{ %spacing
+#' #
```
Comment (lionel-) [inline R/composition.R]: do you know of a better way to ensure that Roxygen leaves some whitespace when it collates the examples together?

**Comment by lionel-** [comment]
thanks for the kind mention in the announcement :)

**Comment by hadley** [comment]
Could you please merge/rebase?

File: R/composition.R (hunk 1 above)
Comment (hadley) [inline R/composition.R]: roxygen2 5.0.0 now keeps by default :)

**Comment by lionel-** [comment]
Here you go.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**